src/test/app/claims/models/claim.ts

```diff
@@ -63,6 +66,20 @@ describe('Claim', () => {
         expect(claim.eligibleForCCJ).to.be.equal(false)
       })
     })
+
+    context('totalAmountTillToday', () => {
+      it('should return the correct amount with interest until today', () => {
+        expect(sampleClaimObj.totalAmountTillToday).to.equal(225)
+      })
+    })
+
+    context('totalAmountTillDateOfIssue', () => {
+      const inPast = MomentFactory.currentDate().subtract(200, 'weeks')
+      sampleClaimObj.createdAt = inPast
+      it('should return the correct amount with interest until date of issue', () => {
+        expect(sampleClaimObj.totalAmountTillDateOfIssue).to.equal(225)
```

Comment:
Did you notice that no matter what date you set as issue date, the value it always the same? Please test all possible combinations.